src/shared/missing-preact-types.d.ts

```diff
@@ -1,7 +1,11 @@
 declare module 'preact' {
   namespace JSX {
     interface HTMLAttributes {
-      decoding?: 'sync' | 'async' | 'auto';
+      decoding?:
+        | 'sync'
+        | 'async'
+        | 'auto'
+        | SignalLike<'sync' | 'async' | 'auto' | undefined>;
```

Comment:
Should this be here?

Reply:
Without that I get this error: `Subsequent property declarations must have the same type.  Property 'decoding' must be of type '"sync" | "async" | "auto" | SignalLike<"sync" | "async" | "auto" | undefined> | undefined', but here has type '"sync" | "async" | "auto" | undefined'`.

Reply:
@jakearchibald After further investigation I found that the error was caused by a package version error. After fixing that, the error is gone, so the code you mentioned